docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,234 @@
+and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
+
+You can then use Briefcase's ``build`` command to compile this into an Android
+APK app file. It will take quite a few minutes the first time, depending on
+your Internet connection speed, and then should take less than 30 seconds every
+successive time.
+
+.. admonition:: Gradle can look stuck
+
+  You may see Gradle stuck on the "CONFIGURING" step for many, many minutes.
+  Gradle is likely busy performing downloads, and you should not interrupt it.
+  This typically only affects the first time an app is built.
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase build android
+  [helloworld] Building Android APK...
+  Starting a Gradle Daemon
+  ...
+  BUILD SUCCESSFUL in 1m 1s
+  28 actionable tasks: 17 executed, 11 up-to-date
+  [helloworld] Built android/Hello World/app/build/outputs/apk/debug/app-debug.apk
+
+Run the app on a virtual device
+===============================
+
+We're now ready to run our application. You can use Briefcase's ``run`` command
+to run the app on an Android device, either physical or virtual. This section
+covers running the app on a virtual Android device.
+
+On Android, Briefcase's ``run`` command requires that you specify which device to run on.
+If you use ``run`` without specifying a device, Briefcase will show a list of
+physical and virtual devices (if any) and offer to create one.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      (beeware-venv) C:\...>briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+Choose your desired device, or choose the option to create a new device. If you create a new
+device, you will have to provide a name (or accept the default of ``beePhone``).
+
+Once you choose a device, you will see two steps of Android booting followed by your app
+launching. First, the Android logo will appear.
+
+.. figure:: ../images/android/tutorial-5-booting.png
+   :alt: Android virtual device booting
+
+   Android virtual device booting
+
+In the second phase, the launcher screen will appear.
+
+.. figure:: ../images/android/tutorial-5-running.png
+   :alt: Android virtual device fully started, on the launcher screen
+
+   Android virtual device fully started, on the launcher screen
+
+Once the phone finishes booting, ``briefcase run`` will automatically begin
+to launch your app. You will first see a splash screen, then your app. If you
```

Comment:
This makes it sound like there's an additional user step, or that ``briefcase run`` has to be re-executed.

We may also want to put in a screen shot of the splash screen, and the actual app running.

Reply:
Fixed in c6bda2f